index.html

```diff
@@ -338,14 +338,26 @@
             <ol>
               <li>Instantiate an <a>installation process</a>.
               </li>
-              <li>Let <a>manifest</a> be the result of <a>obtaining the
+              <li>Let <a>manifest</a> and <a>manifest URL</a> be the result of <a>obtaining the
               manifest</a>.
               </li>
               <li>If <a>obtaining the manifest</a> results in an error, a user
               agent can, at this point, fall back to using the <a>top-level
               browsing context</a>' <code>Document</code>'s metadata to
               populate an <a>installation process</a>' UI.
               </li>
+              <li>If <a>obtaining the manifest</a> succeeds, and the
+              <var>service worker registration</var> of <a>manifest</a> is not
+              <code>undefined</code>, a user agent can at this point
+              <ul>
+                <li>Invoke <a>Start Register</a> with <var>scope</var> and <var>src</var>
+                members of the <var>service worker registration</var>, a new <var>promise</var>,
+                null, <a>manifest URL</a>, <code>"classic"</code> and <code>False</code>,
```

Comment:
For the _client_ param to Start Register, I think the browsing context's Document's relevant settings object should be passed instead of null. The given client is used to queue a task to later to manipulate promise settlement and parse a fetched script resources later.